<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="searchBar"/>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inputToolsButton">
       <property name="text">
        <string>Input Tools</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="sortByLabel">
       <property name="text">
        <string>Sort by:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sortByComboBox">
       <item>
        <property name="text">
         <string>Relevance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Popularity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Part of Speech</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Complexity</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="advancedCheckbox">
       <property name="text">
        <string>Advanced</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="advancedScrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>297</width>
        <height>395</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="exactMatchesOnlyCheckbox">
            <property name="text">
             <string>Exact matches only</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="deinflectAutomaticallyCheckbox">
            <property name="text">
             <string>Deinflect automatically</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="mostCommonLabel">
              <property name="text">
               <string>Within most common:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="mostCommonSpinBox">
              <property name="maximum">
               <number>99999999</number>
              </property>
              <property name="value">
               <number>10000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="fieldsGroupBox">
            <property name="title">
             <string>Fields</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QCheckBox" name="checkBox_5">
               <property name="text">
                <string>Martial Arts</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_6">
               <property name="text">
                <string>Buddhism</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_7">
               <property name="text">
                <string>Chemistry</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_8">
               <property name="text">
                <string>Computers</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_11">
               <property name="text">
                <string>Food</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_12">
               <property name="text">
                <string>Geometry</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_13">
               <property name="text">
                <string>Linguistics</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_14">
               <property name="text">
                <string>Mathematics</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="partsOfSpeechGroupBox">
            <property name="title">
             <string>Parts of Speech</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QCheckBox" name="checkBox_9">
               <property name="text">
                <string>Adjective （形容詞）</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="checkBox_10">
               <property name="text">
                <string>Adjective （形容詞） - 良い class</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWebEngineView" name="contentsView" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
